src/main/java/com/kjetland/dropwizard/activemq/ActiveMQBundle.java

```diff
@@ -167,6 +175,12 @@ public ActiveMQSender createSender(String destination, boolean persistent, Optio
     // This must be used during run-phase
     public <T> ActiveMQReceiverHandler<T> registerReceiver(String destination, ActiveMQReceiver<T> receiver, Class<? extends T> clazz,
                                                            ActiveMQBaseExceptionHandler exceptionHandler) {
+        return registerReceiver(destination, receiver, clazz, exceptionHandler, null);
+    }
+
+    // This must be used during run-phase
+    public <T> ActiveMQReceiverHandler<T> registerReceiver(String destination, ActiveMQReceiver<T> receiver, Class<? extends T> clazz,

```

Comment:
we should add an integration test to check the selectors support (smth very similar to what we have in `ActiveMQReceiverHandlerRedeliveryTest`)